src/CattyTests/DataModel/UserVariableTests.swift

```diff
@@ -70,4 +70,73 @@ final class UserVariableTests: XCTestCase {
         XCTAssertFalse(userVariableA === userVariableCopy)
         XCTAssertTrue(userVariableB === userVariableCopy)
     }
+
+    func testIsEqualToUserVariableForDirectAssignment() {
+        let userVariableA = UserVariable()
+        userVariableA.name = "userVar"
+        userVariableA.value = 56
+
+        var userVariableB = userVariableA
+        XCTAssertTrue(userVariableA.isEqual(to: userVariableB))
+        XCTAssertEqual(userVariableB.value as! Int, 56)
+
+        let userVariableC = UserVariable()
+        userVariableC.name = "userVarC"
+
+        userVariableB = userVariableC
+        XCTAssertFalse(userVariableA.isEqual(to: userVariableB))
+    }
+
+    func testIsEqualToUserVariableForInitWithNameAndIsList() {
+        let userVariableA = UserVariable()
+        userVariableA.name = "userVar"
+        userVariableA.value = 56
+
+        var userVariableB = UserVariable(name: userVariableA.name, isList: userVariableA.isList)
```

Comment:
Thanks for your changes, looks good to me! Our data structure is far away from being ideal, that's why your tests are very important for us to build up a regression test suite before starting with major refactoring. However, the value of such lists is always an array. So could you please add a test or adapt an existing which also tests this method when the value is an array containing (a) several equal elements, and (b) several elements but one different element?